README.md

```diff
@@ -11,6 +11,17 @@
 
 For documentation, visit [https://frint.js.org](https://frint.js.org).
 
+Key characteristics and features include:
+
+* Gives your applications a **structure**
+* **Environment** agnostic (runs in browser, server, and CLI)
+* **Rendering** library agnostic (integrates with React, Vue, and Preact)
+* **Composable** with multiple packages as needed
+* Each package focused on doing **one thing** well only
```

Comment:
1. I think "focused" should become either "focuses" or "is focused"
2. The "...doing one thing well only" part reads weird. Maybe it should be "...doing one thing only and doing it well", or something shorter I can't come up with. Unix authors also couldn't :D https://en.wikipedia.org/wiki/Unix_philosophy#Do_One_Thing_and_Do_It_Well